<launch>

	<include
		file="$(find autonomous)/launch/camera.launch"
	/>

	
	<node
		pkg="autonomous"
		name="image_preprocessing_node"
		type="image_preprocessing_node.py"
		respawn="true"
	/>

	
	<node
		pkg="autonomous"
		name="image_lanedetection_node"
		type="image_lanedetection_node.py"
		respawn="true"
	/>

	
	<node
		pkg="autonomous"
		name="image_stopsign_node"
		type="image_stopsign_node.py"
		respawn="true"
	/>

	
	<node
		pkg="rosserial_python"
		name="serial_node"
		type="serial_node.py"
		args="/dev/ttyACM0"
		respawn="true"
	/>

	<node
		pkg="autonomous"
		name="car_control_node"
		type="car_control_node.py"
		respawn="true"
	/>


</launch>
